Gemfile

```
@@ -7,7 +7,7 @@ source 'https://rubygems.org'
 
 gem 'rails', case SETTINGS[:rails]
              when '6.0'
-               '~> 6.0.3.1'
+               '~> 6.0.4.7'
```

Would it be safer to update to 6.0.3.7
```suggestion
               '~> 6.0.3.7'
```

---

I was aiming to at least pull in this fix https://github.com/rails/rails/security/advisories/GHSA-wh98-p28r-vrc9 along with any other bug and security fixes that have been released since the version we are on.

---

We are on 6.0.3.7 (via packaging) already.